cvat/apps/engine/media_extractors.py

```diff
@@ -58,9 +57,8 @@ def _get_preview(self, obj):
             preview = obj
         preview.thumbnail((128, 128))
 
-        return preview
+        return preview.convert('RGB')
```

Comment:
Why do we need the line? What is the problem?

Reply:
I tried create a task with png RGBA image and got the following error
```
raise IOError("cannot write mode %s as JPEG" % im.mode)
OSError: cannot write mode RGBA as JPEG
```